PKHeX.WinForms/Program.cs

```diff
@@ -93,6 +94,12 @@ private static void CurrentDomain_UnhandledException(object sender, UnhandledExc
                 // Todo: make this translatable
                 ErrorWindow.ShowErrorDialog("An unhandled exception has occurred.\nPKHeX must now close.", ex, false);
             }
+            catch (MissingMethodException)
+            {
+                // Todo: make this translatable
+                MessageBox.Show(".NET Framework 4.6 needs to be installed for this version of PKHeX to run.", "PKHeX Error", MessageBoxButtons.OK, MessageBoxIcon.Stop);
+                Process.Start(@"https://www.microsoft.com/en-us/download/details.aspx?id=48130");
```

Comment:
Think it's better to prevent those not on 4.6 from starting the application instead of try/catching or working unhandled exceptions since we don't know what other issues may arise from continued use.